backend/ethereum/wallet/simple/account.go

```diff
@@ -37,14 +37,15 @@ func (a *Account) Address() wallet.Address {
 }
 
 // SignData is used to sign data with this account.
-func (a *Account) SignData(data []byte) ([]byte, error) {
+func (a *Account) SignData(data []byte) (wallet.Sig, error) {
 	hash := ethwallet.PrefixedHash(data)
 	sig, err := a.SignHash(hash)
 	if err != nil {
 		return nil, errors.Wrap(err, "SignHash")
 	}
 	sig[64] += 27
```

Comment:
Should add a comment that explains what is happening here.

Should use a const instead of `27`. Why is the linter not catching this?

(Could move this part to a central location that is used by the other Ethereum wallet implementations as well.)